files-community/Files
# Feature: Updated the design of the Tags section in the Details Pane

**Resolved / Related Issues**
   Related to #12946 

**Validation**
How did you test these changes?
   1. Open details pane
   2. Edit the tags using the menu flyout

**Screenshots (optional)**
Add screenshots here.
